### PR TITLE
Create question form

### PR DESCRIPTION
**Pull request by desplodur**
Please take a look at my PR

### CODE REVIEW

File: src/components/QuestionForm/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import StyledQuestionForm from './styled';
+
+export default function QuestionForm({onSubmitFunction}) {
```
Comment (pixelass) [inline src/components/QuestionForm/index.js:3]: I'd suggest calling the handler `onSubmit` (it is implied that this is a function)

File: src/pages/index.js [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+	]);
+
+	const createNewQuestion = event => {
+		event.preventDefault();
+		const newQuestion = {
+			id: nanoid(),
+			questionText: event.target.questionTextInput.value,
+		};
+		const newQuestionArray = [...questionArray, newQuestion];
+		setQuestionArray(newQuestionArray);
```
Comment (pixelass) [inline src/pages/index.js:23]:
```suggestion
		setQuestionArray([...questionArray, newQuestion]);
```

Redundant constant assignment. (You can keep it if it is easier for you to understand, but it is not needed here.

**Review by pixelass — COMMENTED** [review]
Feel free to adjust, based on the comments.
They are recommendations, decide which ones you want to adjust.

File: src/pages/index.js [hunk 3]
```diff
@@ -1,28 +1,44 @@
...
+		console.log(questionArray);
```
[the remark this answers is not in the document]
Comment (desplodur) [reply src/pages/index.js:23]: I noticed that already. It was just a short thought testing and will be removed. :)

File: src/components/QuestionForm/QuestionForm.spec.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	it('should return the input field value', async () => {
+		const onSubmit = jest.fn(event => {
+			event.preventDefault();
+			fireEvent.change(inputField, {target: {value: 'test'}});
+		});
+		render(<QuestionForm onSubmitFunction={onSubmit} />);
+		const submitButton = screen.getByRole('button', {type: 'submit'});
+		const inputField = screen.getByRole('textbox', {type: 'text'});
+		await fireEvent.click(submitButton);
+		expect(inputField.value).toBe('test');
```
[the remark this answers is not in the document]
Comment (desplodur) [reply src/components/QuestionForm/QuestionForm.spec.js:37]: I was so proud of my solution but you are right 😁

i deleted the whole onSubmit / jest.fn part

File: src/components/QuestionForm/index.js (hunk 1 above)
Comment (desplodur) [reply src/components/QuestionForm/index.js:3]: done

**Review by pixelass — APPROVED** [review]

**Pull request merged**